src/test/java/kitchenpos/products/tobe/domain/Fixtures.java

```diff
@@ -0,0 +1,27 @@
+package kitchenpos.products.tobe.domain;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+
+public class Fixtures {
+
+    public static Product product(final UUID id, final String name, final BigDecimal price) {
```

Comment:
Fixture는 자주 사용되거나 의미를 가지는 테스트 객체를 명확히 정의하는 편이 좋아요. 
제가 사용하는 방식도 공유드릴게요. 
```suggestion
    public static Product DEFAULT() {
        return product(UUID.randomUUID(), "후라이드", BigDecimal.valueOf(16_000L));
    }
    
    public static Product FRIED_CHICKEN(final UUID id) {
        return product(id, "후라이드", BigDecimal.valueOf(16_000L));
    }
    
    public static Product WITH_NAME(fina String name) {
        return product(UUID.randomUUID(), name, BigDecimal.valueOf(16_000L));
    }
```
위와 같이 특정 의미를 가지는 픽스쳐와 자주 변동 되는 픽스쳐, 자주 사용하는 픽스쳐를 정의하는 편입니다 ㅎㅎ